.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+# Description
+Briefly describe the problem or reason for the changes implemented in the PR
+
+# Trac ticket
+Link to the accepted Trac ticket for this PR. All PRs except docs or tests changes need an accepted ticket. New features need community consensus.
+
+# Tests
+Briefly describe any tests you have added or updated to test the changes introduced by the PR
```

Comment:
Is that generally expected? I feel like having the tests in the checklist is good enough – asking people to describe the tests they've written feels like a sure-fire way to get the paragraph deleted entirely.

Reply:
I agree 👍

Additionally when we're also draining people's cognitive resources by adding too much stuff to read so repetitive things should be removed.

Less is more :)